static/app/views/replays/detail/header/replayDetailsUserBadge.tsx

```diff
@@ -160,7 +202,9 @@ const DisplayHeader = styled('div')`
 function Live() {
   return (
     <Flex align="center">
-      <LiveText bold>{t('LIVE')}</LiveText>
+      <LiveText bold data-test-id="live-badge">
```

Comment:
Do we need to use styled for LiveText since `Text` supports a `variant` for color?  https://sentry.sentry.io/stories/typography/text#variants